tests/nn/initializers_test.py

```diff
@@ -107,3 +110,13 @@ def forward(self, inputs):  # pylint: disable=arguments-differ
 
         for parameter in list(model.linear2.parameters()):
             assert torch.equal(parameter.data, torch.ones(parameter.size()) * 7)
+
+    def test_block_orthogonal_can_initialize(self):
+        # TODO(Mark): This is a rubbish test. See if Matt/Joel have a better idea.
+        tensor = torch.zeros([10, 6, 8])
+        block_orthogonal(tensor, [5, 2, 2])
```

Comment:
naively, I want to say can't you check that the blocks are all orthogonal?  iterate over them and then for each one do

```
x = block.numpy()
assert np.all(np.isclose(x.T @ x, np.eye(size)))
```

?

I don't know exactly what `torch.nn.init.orthogonal` is doing on higher-dimensional tensors, though.